app/javascript/react/screens/App/Overview/components/Migrations/ScheduleMigrationButton.js

```diff
@@ -14,13 +15,14 @@ const ScheduleMigrationButton = ({
   plan,
   isMissingMapping
 }) => {
-  const migrationScheduled = (plan.schedules && plan.schedules[0].run_at.start_time) || 0;
-  const staleMigrationSchedule = new Date(migrationScheduled).getTime() < Date.now();
+  const { migrationScheduled, staleMigrationSchedule, migrationStarting } = getPlanScheduleInfo(plan);
+  const showScheduleButton = staleMigrationSchedule && !migrationStarting;
```

Comment:
This would allow a user to edit a schedule that was never run, correct?

Is this scenario actually possible @AparnaKarve?  If so, are there any possible negative side effects of editing a stale schedule to run in the future?

Reply:
We do allow editing a schedule that was never run, in other areas of the UI (in Configuration -> Schedules)

Can't think of any possible negative effects of editing a stale schedule to run in the future.
I think this is good!

Reply:
The existing code already allowed editing a schedule that was never run, I believe.